scipy/stats/tests/test_distributions.py

```diff
@@ -5003,6 +5003,13 @@ def test_extreme_entropy(self, a, b, ref, tol):
         #     return float(entropy)
         assert_allclose(stats.beta(a, b).entropy(), ref, rtol=tol)
 
+    def test_entropy_broadcasting(self):
+        # gh-23127 reported that the entropy method of the beta
+        # distribution did not broadcast correctly.
+        beta_dist = stats.make_distribution(stats.beta)
+        b = beta_dist(a=np.zeros(10), b=np.ones(10))
+        b.entropy()
```

Comment:
I've been using camelcase distribution names as the output of `make_distribution` and capital names for the random variables once they're instantiated. Also, This is probably overkill, but I wanted to try it, so I thought I might as well include it here.
```suggestion
        Beta = stats.make_distribution(stats.beta)
        a = np.asarray([5e6, 100, 1e9, 10])
        b = np.asarray([5e6, 1e9, 100, 20])
        res = Beta(a=a, b=b).entropy()
        ref = np.asarray([Beta(a=a[0], b=b[0]).entropy(),
                          Beta(a=a[1], b=b[1]).entropy(),
                          Beta(a=a[2], b=b[2]).entropy(),
                          Beta(a=a[3], b=b[3]).entropy()])
        assert_allclose(res, ref)
```
It might be good to at least check the shape of the output, though.